worlds/oot/Options.py

```diff
@@ -1,9 +1,34 @@
 import typing
+import random
 from Options import Option, DefaultOnToggle, Toggle, Range, OptionList, DeathLink
 from .LogicTricks import normalized_name_tricks
 from .ColorSFXOptions import *
 
 
+class TrackRandomRange(Range):
```

Comment:
A bit sad to see this be OoT specific. I think Violet wanted something similar for Witness.

Reply:
I couldn't think of a good way to make this work for all option types besides overhauling Generate.py, and I'd rather do that in a separate PR. If that happens, this randomization tracking will be extended to all options that support weighting.